cmd/loadtest/loadtest.go

```diff
@@ -396,8 +405,33 @@ func runLoadTest(ctx context.Context) error {
 		overallTimer = new(time.Timer)
 	}
 
-	// Dial the Ethereum RPC server.
-	rpc, err := ethrpc.DialContext(ctx, *inputLoadTestParams.RPCUrl)
+	// connLimit is the value we'll use to configure the connection limit within the http transport
+	connLimit := 2 * int(*inputLoadTestParams.Concurrency)
+	// Most of these transport options are defaults. We might want to make this configurable from the CLI at some point.
+	// The goal here is to avoid opening a ton of connections that go idle then get closed and eventually exhausting
+	// client-side connections.
+	transport := &http.Transport{
+		DisableKeepAlives:      false,
+		DisableCompression:     false,
+		MaxIdleConns:           connLimit,
+		MaxIdleConnsPerHost:    connLimit,
+		MaxConnsPerHost:        connLimit,
```

Comment:
I was actually just thinking about this in another context. The general question being, is worth explicitly setting a value that has a default that you like already? The goal being to guard against future changes in the default?

In this case, it's probably overkill because maybe these values aren't going to change.

I think the only reason I have the whole thing is because I wasn't sure which once I would need to change in order to address the issue. After just changing those, it worked properly haha.. 

I'll push another commit to drop the rest of the fields shortly

Reply:
Yeah makes sense, I'm open to leave the fields or drop them :)